feat(Modal/Popover/Slideover): add `close-prevented` event

rdjanuar opened the pull request:
<!---
☝️ PR title should follow conventional commits (https://conventionalcommits.org)
-->

### 🔗 Linked issue

Resolves https://github.com/nuxt/ui/issues/3126

### ❓ Type of change

<!-- What types of changes does your code introduce? Put an `x` in all the boxes that apply. -->

- [x] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [x] 👌 Enhancement (improving an existing functionality)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

<!-- Describe your changes in detail -->
<!-- Why is this change required? What problem does it solve? -->

### 📝 Checklist

<!-- Put an `x` in all the boxes that apply. -->
<!-- If your change requires a documentation PR, please link it appropriately -->
<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

rdjanuar committed:
feat(Modal): new event close-prevented

benjamincanac reviewed (commented):
I guess this should be done on the `Slideover` and `Popover` components that have the same behavior with a `dismissible` prop.

benjamincanac committed:
chore(release): v3.0.0-beta.3

benjamincanac committed:
docs(deps): update `@nuxt/ui-pro`

ivanmaxlogiudice committed:
feat(Input/Textarea): allow `null` value in model (#3415)

benjamincanac committed:
chore(templates): add tsdoc on `ui` app config key

benjamincanac committed:
docs(installation): improve continuous releases section

renovate[bot] committed:
chore(deps): update all non-major dependencies (v3) (#3489)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

benjamincanac committed:
chore(LinkBase): update types for `nuxt@3.16`

renovate[bot] committed:
chore(deps): update dependency tailwind-variants to v1 (v3) (#3499)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Benjamin Canac <canacb1@gmail.com>

benjamincanac committed:
fix(Button): missing import

Related to nuxt/ui#3417

renovate[bot] committed:
chore(deps): update all non-major dependencies (v3) (#3497)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update nuxt framework to ^3.16.0 (v3) (#3484)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Benjamin Canac <canacb1@gmail.com>
Co-authored-by: Pooya Parsa <pooya@pi0.io>

benjamincanac committed:
docs(app): add missing `cache-key` on mdc

benjamincanac committed:
chore(deps): update `@nuxt/ui-pro`

ivanmaxlogiudice committed:
fix(Form): input blur validation on submit (#3504)

renovate[bot] committed:
chore(deps): update vueuse monorepo to v13 (v3) (major) (#3512)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

benjamincanac committed:
chore(renovate): ignore `vaul-vue`

benjamincanac committed:
chore(deps): remove `happy-dom` resolution

benjamincanac committed:
chore(vitest): improve config to ignore docs `.c12`

renovate[bot] committed:
chore(deps): update all non-major dependencies (v3) (#3502)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): lock file maintenance (v3) (#3508)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

benjamincanac committed:
chore(Pagination): remove `vue-router` import

benjamincanac committed:
chore(deps): add `vue` / `vue-router` as dependencies

genu committed:
docs(migration): describe `useToast` and `useOverlay` (#3509)

Co-authored-by: Benjamin Canac <canacb1@gmail.com>

genu committed:
docs(migration): fix typo (#3516)

hywax committed:
feat(useLocale): handle generic messages (#3100)

Co-authored-by: hywax <me@hywax.space>
Co-authored-by: Benjamin Canac <canacb1@gmail.com>

benjamincanac committed:
chore(deps): update `@nuxt/ui-pro`

benjamincanac committed:
playground-vue: add logo

Justineo committed:
docs(figma): improve figma to code section (#3520)

renovate[bot] committed:
chore(deps): update all non-major dependencies (v3) (#3518)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

benjamincanac committed:
fix(vue): prevent calling `useHead` in colors

romhml committed:
feat(Form): global errors (#3482)

Co-authored-by: Benjamin Canac <canacb1@gmail.com>

renovate[bot] committed:
chore(deps): update tailwindcss to ^4.0.13 (v3) (#3526)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update all non-major dependencies (v3) (#3521)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

benjamincanac committed:
chore(release): v3.0.0-beta.4

benjamincanac committed:
docs: rename `dev` to `v2`

benjamincanac committed:
chore(package): update `release` command

benjamincanac committed:
chore(release): v3.0.0

atinux committed:
docs(installation): rename template to ui

benjamincanac committed:
chore(github): use `ui2.nuxt.com`

atinux committed:
docs: fix og image

benjamincanac committed:
chore(github): use `ui.nuxt.com` url in v3

benjamincanac committed:
docs: update for `v3.0.0` release

benjamincanac committed:
chore(deps): update `@nuxt/ui-pro`

benjamincanac committed:
chore: use `ui.nuxt.com` url

benjamincanac committed:
chore(vercel): remove useless file

atinux committed:
docs(pro): launch offer

atinux committed:
docs: add ui pro banner

benjamincanac committed:
chore(vercel): put back config

maximepvrt committed:
chore(readme): fix npm badge (#3540)

renovate[bot] committed:
chore(deps): update tailwindcss to ^4.0.14 (v3) (#3553)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

benjamincanac committed:
chore(github): add integration workflow

benjamincanac committed:
chore(github): update integration workflow

benjamincanac committed:
chore(github): update integration workflow

benjamincanac committed:
chore(github): update integration workflow

benjamincanac committed:
docs(pro): disable launch offer

jwoertink committed:
docs(migration): fix typo (#3563)

NilsEvers committed:
fix(types): add missing export for Icon (#3568)

Barbapapazes committed:
docs(pro): update barbapapazes description (#3577)

Kheuval committed:
fix(Table): allow links to be opened when @select is used (#3580)

benjamincanac committed:
docs(nuxt.config): fix local extends

Related to https://github.com/nuxt/nuxt/issues/31311

benjamincanac committed:
docs: clean badges

harlan-zw committed:
fix(vue): missing unhead context (#3589)

Co-authored-by: Benjamin Canac <canacb1@gmail.com>

romhml committed:
fix(unplugin): include `@compodium/examples` in auto-imports paths (#3585)

benjamincanac committed:
docs(app): put back carbon ads

renovate[bot] committed:
chore(deps): update all non-major dependencies (v3) (#3551)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Benjamin Canac <canacb1@gmail.com>

romhml committed:
docs(installation): improve vscode recommendations (#3592)

Co-authored-by: Benjamin Canac <canacb1@gmail.com>

benjamincanac committed:
chore(colors): put back `tagPriority`

Partial revert of #3589

renovate[bot] committed:
chore(deps): lock file maintenance (v3) (#3582)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

harlan-zw committed:
docs: broken 404 pages (#3598)

benjamincanac committed:
chore(components): add eol in script tag to fix syntax highlight

Rizwan-malek committed:
docs(migration): improve `useOverlay` section (#3601)

benjamincanac committed:
fix(useLocale): unique symbol to use in `@nuxt/ui-pro` (#3603)

danielroe committed:
fix(module): mark functions used in exports as pure (#3604)

romhml committed:
chore(Form): improve TSDoc (#3619)

romhml committed:
refactor(Form)!: drop explicit support for `zod` and `valibot` (#3617)

benjamincanac committed:
docs(Banner): allow to close

benjamincanac committed:
docs(app): improve safelist

benjamincanac committed:
chore(deps): update `@nuxt/ui-pro`

benjamincanac committed:
docs(app): improve stars lazy loading

benjamincanac committed:
chore(github): update integration workflow

benjamincanac committed:
chore(github): update integration workflow

benjamincanac committed:
chore(github): update integration workflow

Norbiros committed:
fix(ContextMenu/DropdownMenu): remove `any` from `proxySlots` (#3623)

renovate[bot] committed:
chore(deps): update nuxt framework to ^3.16.1 (v3) (#3626)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

benjamincanac committed:
docs(app): remove hydrate attributes on stars

benjamincanac committed:
docs(Header): remove extraneous prop on navigation menu

benjamincanac committed:
fix(Modal/Slideover/Toast): prevent unnecessary close instantiation

benjamincanac committed:
feat(components): handle events in `content` prop

benjamincanac committed:
test: update snapshots

atinux committed:
docs: add redirects from v2 structure (#3634)

ArbazIrshad committed:
feat(locale): add Urdu language (#3611)

UlugbekNG committed:
feat(locale): add Uzbek language (#3548)

Co-authored-by: Benjamin Canac <canacb1@gmail.com>

markmelnic committed:
feat(locale): add Romanian language (#3587)

mickaelchanrion committed:
feat(locale): add Catalan language (#3550)

Co-authored-by: Benjamin Canac <canacb1@gmail.com>

halwesit committed:
feat(locale): add Central Kurdish language (#3566)

Co-authored-by: Benjamin Canac <canacb1@gmail.com>

renovate[bot] committed:
chore(deps): update all non-major dependencies (v3) (#3591)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

sandros94 committed:
fix(module): handle tailwindcss import without `theme(static)` (#3630)

Co-authored-by: Benjamin Canac <canacb1@gmail.com>

benjamincanac committed:
chore(github): update integration workflow

benjamincanac committed:
chore(github): update integration workflow

benjamincanac committed:
fix(RadioGroup): handle `disabled` on items

Resolves nuxt/ui-pro#911

benjamincanac committed:
chore(github): update integration workflow

benjamincanac committed:
chore(github): update integration workflow

benjamincanac committed:
chore(github): update integration workflow

benjamincanac committed:
chore(deps): update `vaul-vue`

benjamincanac committed:
docs(deps): use only one motion library

benjamincanac committed:
chore(github): remove integration env

benjamincanac committed:
chore(Drawer): put back `shrink-0` on handle

renovate[bot] committed:
chore(deps): update tailwindcss to ^4.0.15 (v3) (#3638)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

ac-shadow committed:
fix(Calendar): grey out days outside of displayed month (#3639)

benjamincanac committed:
chore(github): update integration workflow (#3643)

benjamincanac committed:
chore(release): v3.0.1

benjamincanac committed:
chore(github): enable ui pro integration

benjamincanac committed:
chore(deps): move `@standard-schema/spec` to `dependencies`

benjamincanac committed:
fix(defineLocale/defineShortcuts): remove `@__NO_SIDE_EFFECTS__`

benjamincanac committed:
fix(Drawer): remove `fadeFromIndex` prop proxy

benjamincanac committed:
fix(Switch): prevent transition on focus outline

benjamincanac committed:
fix(Tabs): remove `focus:outline-hidden` class

benjamincanac committed:
fix(Button): use `focus:outline-none` instead of `focus:outline-hidden`

Resolves #3658

benjamincanac committed:
chore(defineLocale): put back `@__NO_SIDE_EFFECTS__`

renovate[bot] committed:
chore(deps): update all non-major dependencies (v3) (#3636)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Benjamin Canac <canacb1@gmail.com>

benjamincanac committed:
chore(docs/playground): add `vite.optimizeDeps`

renovate[bot] committed:
chore(deps): lock file maintenance (v3) (#3662)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

Commit:
feat(locale): add Armenian language (#3664)

benjamincanac committed:
chore(github): improve module workflow

romhml committed:
chore(deps): declare form validation libraries as `peerDependencies` (#3666)

benjamincanac committed:
playground(deps): add `zod`

benjamincanac committed:
chore(renovate): ignore `peerDependencies`

benjamincanac committed:
chore(deps): update `peerDependencies`

benjamincanac committed:
chore(deps): remove `typescript` resolution

atinux committed:
docs: add missing redirect

benjamincanac committed:
fix(NavigationMenu): add `z-index` on viewport

Resolves #3654

benjamincanac committed:
docs(index): improve marquees performances

renovate[bot] committed:
chore(deps): update devdependency wrangler to v4 (v3) (#3554)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

benjamincanac committed:
docs(nuxt.config): update `vite.optimizeDeps`

benjamincanac committed:
chore(github): update `stale` to close `needs reproduction`

benjamincanac committed:
fix(Link): properly pick all `aria-*` & `data-*` attrs

HugoRCD committed:
feat(Calendar): allow year and month buttons styling (#3672)

benjamincanac committed:
fix(Link): proxy `onClick`

Resolves #3631

sandros94 committed:
fix(components): improve generic types (#3331)

Co-authored-by: Benjamin Canac <canacb1@gmail.com>

benjamincanac committed:
fix(Link): prevent `active="true"` binding on html

benjamincanac committed:
fix(Link): handle `aria-current` like `NuxtLink` / `RouterLink`

BobbieGoede committed:
docs(i18n): remove `next` tag from `@nuxtjs/i18n` installation (#3675)

benjamincanac committed:
fix(Container): add `w-full` class

HugoRCD committed:
docs: improve lighthouse score and accessibility (#3673)

renovate[bot] committed:
chore(deps): update tailwindcss to ^4.0.16 (v3) (#3682)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update all non-major dependencies (v3) (#3667)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

stijns96 committed:
docs(accordion): add drag and drop example (#3684)

Co-authored-by: Benjamin Canac <canacb1@gmail.com>

romhml committed:
fix(FormField): add `help` to `aria-describedby` attribute (#3691)

renovate[bot] committed:
chore(deps): update tailwindcss to ^4.0.17 (v3) (#3690)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

romhml committed:
fix(Form): clear dirty state after submit (#3692)

renovate[bot] committed:
chore(deps): update all non-major dependencies (v3) (#3695)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

benjamincanac committed:
docs(slug): add purchase and affilitate in community links

benjamincanac committed:
feat(Table): add `empty` prop

benjamincanac committed:
fix(Table): wrong condition on `caption` slot

benjamincanac committed:
chore(CommandPalette): improve `placeholder` tsdoc

benjamincanac committed:
chore(deps): update `@nuxt/ui-pro`

benjamincanac committed:
chore(github): prevent running module `nuxt-ui-pro` job on forks

benjamincanac committed:
test(Table): update snapshots

benjamincanac committed:
docs(SupportedLanguages): reorder mapping

benjamincanac committed:
fix(InputMenu/SelectMenu): empty search results

benjamincanac committed:
fix(InputMenu): reset `searchTerm` on `update:open`

Resolves #3620

malik-jouda committed:
playground: improve rtl (#3707)

benjamincanac committed:
fix(CommandPalette): use `group.id` as key

benjamincanac committed:
chore(deps): update `@nuxt/ui-pro`

zAlweNy26 committed:
fix(types): add missing export for ButtonGroup (#3709)

Lionad-Morotar committed:
fix(Avatar): proxy `$attrs` to default slot (#3712)

benjamincanac committed:
fix(vue): mock `nuxtApp.hooks` & `useRuntimeHook`

genu committed:
fix(useOverlay): refine `open` method type to infer close emit return type (#3716)

renovate[bot] committed:
chore(deps): update all non-major dependencies (v3) (#3698)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

benjamincanac committed:
playground(app): add title

benjamincanac committed:
chore(release): v3.0.2

benjamincanac committed:
feat(PinInput): add `autofocus` / `autofocus-delay` props

Resolves #3717

benjamincanac committed:
chore(tsconfig): ignore `playground-vue`

benjamincanac committed:
docs(ThemePicker): wrong neutral color

Norbiros committed:
fix(ContextMenuContent/DropdownMenuContent): remove unwanted `any` (#3741)

renovate[bot] committed:
chore(deps): lock file maintenance (v3) (#3746)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update all non-major dependencies (v3) (#3722)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

zAlweNy26 committed:
fix(InputMenu/SelectMenu): correctly call `onSelect` events (#3735)

robinsimonklein committed:
feat(InputNumber): add support for `stepSnapping` & `disableWheelChange` props (#3731)

renovate[bot] committed:
chore(deps): update all non-major dependencies (v3) (#3755)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

romhml committed:
feat(RadioGroup): add `card` and `table`  variants (#3178)

Co-authored-by: Benjamin Canac <canacb1@gmail.com>

benjamincanac committed:
chore(README): add contribution guide

Resolves #3757

benjamincanac committed:
fix(InputMenu): emit `change` on multiple item removal

Fixes #3756

malik-jouda committed:
fix(ContextMenu/DropdownMenu): handle RTL mode (#3744)

benjamincanac committed:
chore(cli): fix `tv` import

benjamincanac committed:
chore(Input/InputNumber/PinInput/Textarea): clean functions order

benjamincanac committed:
feat(Textarea): add `autoresize-delay` prop

Resolves #3730

benjamincanac committed:
feat(Textarea): add `icon`, `loading`, etc. props to match Input

benjamincanac committed:
feat(Textarea): add `resize-none` class with `autoresize` prop

renovate[bot] committed:
chore(deps): update nuxt framework to ^3.16.2 (v3) (#3769)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

benjamincanac committed:
docs(drawer): add responsive example

Resolves #3772

benjamincanac committed:
docs: add team page

benjamincanac committed:
docs(app): improve links

HugoRCD committed:
docs: add showcase page (#3659)

Co-authored-by: Benjamin Canac <canacb1@gmail.com>

benjamincanac committed:
docs(app): improve images lazy loading

renovate[bot] committed:
chore(deps): update tailwindcss to ^4.1.1 (v3) (#3770)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

benjamincanac committed:
docs(radio-group): add badge on unreleased props

Resolves #3765, resolves nuxt/ui-pro#952

benjamincanac committed:
docs(textarea): add badge on unreleased props

sandros94 committed:
fix(InputMenu/SelectMenu): support arbitrary `value` (#3779)

benjamincanac committed:
docs(showcase): update

Gui-Fernandes21 committed:
docs(radio-group): items only accept strings or numbers (#3781)

OrbisK committed:
docs(Header): add GitHub link in mobile menu (#3721)

benjamincanac committed:
fix(Stepper): ui prop override on `icon` and `content` slots

Resolves #3785

renovate[bot] committed:
chore(deps): update tailwindcss to ^4.1.2 (v3) (#3787)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update all non-major dependencies (v3) (#3768)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>
Co-authored-by: Benjamin Canac <canacb1@gmail.com>

benjamincanac committed:
fix(NavigationMenu): add `sm:w-auto` content slot

Resolves #3788

vachmara committed:
docs(showcase): fix contrast on light mode (#3790)

benjamincanac committed:
docs(showcase): update items

benjamincanac committed:
chore(NavigationMenu): remove slots types in `createReusableTemplate`

howbizarre committed:
feat(locale): add Bulgarian language (#3783)

Co-authored-by: Benjamin Canac <canacb1@gmail.com>

benjamincanac committed:
docs(app): clean seo metas

benjamincanac committed:
chore(module): update metas

benjamincanac committed:
chore(deps): update `@nuxt/module-builder` (#3799)

Co-authored-by: Daniel Roe <daniel@roe.dev>

benjamincanac committed:
chore(module): improve `meta`

renovate[bot] committed:
chore(deps): update tailwindcss to ^4.1.3 (v3) (#3798)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

renovate[bot] committed:
chore(deps): update all non-major dependencies (v3) (#3800)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

benjamincanac committed:
chore(Textarea): apply same styles as Input

Partial revert of https://github.com/nuxt/ui/commit/cb193f1d25b5c73ca03dcf10864800350dd1c290

benjamincanac committed:
fix(CommandPalette): increase input font size to avoid zoom

benjamincanac committed:
feat(InputMenu/SelectMenu): handle `resetSearchTermOnSelect`

Resolves #3782

benjamincanac committed:
docs(migration): describe `onClick` change

Resolves #3825

benjamincanac committed:
fix(Carousel): move arrows inside container on mobile

Resolves #3813

benjamincanac committed:
fix(InputMenu/SelectMenu): prevent `disabled` items to be selected

Resolves #3474

benjamincanac committed:
feat(Select): handle `onSelect` field in items

benjamincanac committed:
fix(CommandPalette): prevent hover background on disabled items

renovate[bot] committed:
chore(deps): update all non-major dependencies (v3) (#3806)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

benjamincanac committed:
fix(CommandPalette): consistent alignement with other components

danielroe committed:
chore(deps): bump `vue-sfc-transformer` (#3822)

benjamincanac committed:
chore(package): export utils

benjamincanac committed:
docs(showcase): add `learnvue.co`

benjamincanac committed:
docs(form): improve types

sandros94 committed:
fix(Tree): simplify reusable template types (#3836)

benjamincanac committed:
docs(form): improve joi example

genu committed:
feat(OverlayProvider)!: return an overlay instance from `.open()` (#3829)

benjamincanac committed:
chore(Textarea): put back styles

HugoRCD committed:
docs(components): add new og images (#3824)

sandros94 committed:
chore: run test suite on windows (#3479)

Co-authored-by: Benjamin Canac <canacb1@gmail.com>

sandros94 committed:
chore: simplify theme imports in dev (#3851)

benjamincanac committed:
docs: prepare for chat components (#3844)

renovate[bot] committed:
chore(deps): update all non-major dependencies (v3) (#3830)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

SukkaW committed:
docs(prettier): load from `jsdelivr.net` instead of `unpkg.com` (#3575)

Akryum committed:
feat(Tabs): add `list-leading` and `list-trailing` slots (#3837)

Co-authored-by: Benjamin Canac <canacb1@gmail.com>

benjamincanac committed:
docs(form): display theme

Resolves #3752

HugoRCD committed:
docs(components): update og images (#3859)

avenikolay committed:
feat(locale): add Tajik language (#3850)

jwoertink committed:
docs(components): add missing `children` field in items (#3846)

benjamincanac committed:
fix(components): refactor types after `@nuxt/module-builder` upgrade (#3855)

maximepvrt committed:
feat(Form): export loading state (#3861)

benjamincanac committed:
chore(deps): update `@nuxt/ui-pro`

benjamincanac committed:
fix(types): handle `ClassValue` in `ui` prop

Resolves #3860

benjamincanac commented:
@rdjanuar Would you mind fixing the conflicts? 😬

rdjanuar commented:
i made a new pull request u can check here https://github.com/nuxt/ui/pull/3958. @benjamincanac
